### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by gabryelreyes**
Release of the documentation

Hosted pages can be seen at https://ludubies.github.io/branchdoc_io_test/ , as they are not deployed here yet (only on push to main)

Another option is building Sphinx locally. Requirements: NodeJS (v20) and Python (v3.11).

### CODE REVIEW

File: README.md
Comment (LuDubies) [inline README.md:1]: line 5: needs . at the end
line 14: "...in your sphinx powered..."

**Review by LuDubies — CHANGES_REQUESTED** [review]
Fix REAMDME errors

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Branchdoc
+
+[![documentation](https://github.com/NewTec-GmbH/gitWorkInstructions/actions/workflows/documentation.yml/badge.svg)](https://github.com/NewTec-GmbH/gitWorkInstructions/actions/workflows/documentation.yml)
+
+This is the documentation for the Branchdoc. You can find the resulting GitHub Pages in [here](https://newtec-gmbh.github.io/gitWorkInstructions/)
```
Comment (BlueAndi) [inline README.md:5]: Branchdoc?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* [sphinx-rtd-dark-mode](https://github.com/MrDogeBro/sphinx_rtd_dark_mode) - Adds a toggleable dark mode to the Read the Docs theme for Sphinx. - MIT License
+* [sphinxcontrib-mermaid](https://github.com/mgaitan/sphinxcontrib-mermaid) - Mermaid diagrams in yours sphinx powered docs. - BSD-2 License
+* [sphinx-copybutton](https://github.com/executablebooks/sphinx-copybutton) - Add a "copy" button to code blocks in Sphinx. - MIT License
+* [sphinx-togglebutton](https://github.com/executablebooks/sphinx-togglebutton) - Show and hide content with a button in Sphinx. - MIT License
+* [myst-parser](https://github.com/executablebooks/MyST-Parser) - An extended commonmark compliant parser, with bridges to docutils/sphinx. - MIT License
+* [mermaid-js/mermaid-cli](https://github.com/mermaid-js/mermaid-cli) - Command line tool for the Mermaid library. - MIT License
+
+## Issues, Ideas And Bugs
+
+If you have further ideas or you found some bugs, great! Create an [issue](https://github.com/NewTec-GmbH/pyPolarionCli/issues) or if you are able and willing to fix it by yourself, clone the repository and create a pull request.
```
Comment (BlueAndi) [inline README.md:22]: Link wrong

File: README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## License
+
+The whole source code is published under [BSD-3-Clause](https://github.com/NewTec-GmbH/pyPolarionCli/blob/main/LICENSE).
```
Comment (BlueAndi) [inline README.md:26]: Link wrong

File: doc/git/git.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+## Using Git
+
+To learn more about using Git, you can visit this [cool website](https://learngitbranching.js.org/).
+
+## Ignore Files
+
+You do not always want to commit everything on your folder, for example, lots of automatically-generated documentation.
+This is where the `.gitignore` file comes into play.
+
+[Here](https://github.com/NewTec-GmbH/template_python/blob/4d2acf95ab71a20211560b5fe9471267ede13582/.gitignore) is an example of how a common ignore file looks like for a Python project. You can add or remove lines depending on your use case but you generally want to ignore any file or folder that can be generated.
```
Comment (BlueAndi) [inline doc/git/git.md:94]: Will this kind of links be still valid all the time?

**Review by LuDubies — APPROVED** [review]

**Review by BlueAndi — APPROVED** [review]

**Review by BlueAndi — APPROVED** [review]

**Pull request merged**